infrastructure/swagger/02_info.yaml

```diff
@@ -12,23 +12,32 @@ info:
   description: |
     ## Overview
 
-    API to configure internal systems to allow a Connecting Party to connect. A source of information about Connecting Parties (systems that connect to NHSE) that helps provide system identity attributes that ensure NHSE services know what systems they are connecting to and the legal entities they are sharing data with. This service is driven by APIs that enable self-service for activities such as certificate management and environment access. It also is a store of end-to-end onboarding/integration data and a catalogue of endpoints.
+    Use this API to access the Connecting Party Manager (CPM) service - an internal service for the creation and management of product IDs.
+
+    This API only has a single access mode, application-restricted - signed JWT authentication, through which you may:
+
+    * Create a product team
+    * Create a product
+    * Retrieve details about a product team
+    * Retrieve details about a product
+    * Retrieve a list of all products associated wit a team or all products associated with an organisation, grouped by product team
+    * Delete a product team
+    * Delete a product
 
     ## Who can use this API
 
-    This API can only be used where there is a legal basis to do so. Make sure you have a valid use case before you go too far with your development. You must demonstrate you have a valid use case as part of digital onboarding. Connecting Parties must have an appointed Clinical Safety Officer and undertake a Clinical Safety Assessment.
+    Currently this API is for [internal use only](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses).
 
     ## API status and roadmap
 
-    This API is in [development](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses).
+    This API is in development, meaning:
```

Comment:
Doesnt linking to the development status make sense rather than us explaining it out?

Reply:
This is Steve's change